clang/include/clang/AST/ASTConcept.h

```diff
@@ -140,15 +142,15 @@ class ConceptReference {
   NamedDecl *FoundDecl;
 
   /// \brief The concept named.
-  ConceptDecl *NamedConcept;
+  TemplateDecl *NamedConcept;
```

Comment:
It seems weird for a type named `ConceptReference` to not refer to a `ConceptDecl` anymore